iosApp/iosApp/Pages/Favorites/EditFavoritesPage.swift

```diff
@@ -17,7 +17,11 @@ struct EditFavoritesPage: View {
     @State var favoritesVMState: FavoritesViewModel.State = .init()
     @State var favoritesState: [RouteStopDirection: Bool] = [:]
 
-    let errorBannerVM: ErrorBannerViewModel
+    let errorBannerVM: IErrorBannerViewModel
+
+    // Needed in addition to the VM for compatibility with fetchApi
+    let errorBannerRepository = RepositoryDI().errorBanner
```

Comment:
This is my least favorite change in this PR. I could expose the repo through the errorBannerVM, but would rather not and eventually have more places use the VM directly.